customersupport/customersupport_tests.py

```diff
@@ -0,0 +1,153 @@
+import unittest
+import requests
+import requests_mock
+from config import SALES_URL, HR_URL
+from customersupport.wrappers import sales
+from customersupport.models import Employee
+from customersupport.wrappers import hr, mocked_responses
+from customersupport.wrappers import hr as hr_wrapper
+
+
+class Stubber:
+    def stubMock(m,json):
+        if json is None:
+            return m.get(hr.giveURL(0), json = mocked_responses.hr_get_employee )
+        else:
+            return m.get('http://vm343a.se.rit.edu:8080/employee?employee_id=0', json=json)
+class StubberSalesCustomer:
+    def stubMock(m, json=None):
+        json = {'customers': [{'customerId': 2, 'firstName': 'Joe', 'lastName': 'Jefferson', 'email': 'therealjoe@joemail.com', 'phone': '1231231234'}]}
+        if json is None:
+            return m.get(SALES_URL, json=mocked_responses.sales_search_customer)
+        else:
+            return m.get('http://vm343c.se.rit.edu/api/customer?firstName=Joe',json=json)
+class StubberSalesOrder:
```

Comment:
Please put a newline (or two, officially) above each of these classes for readability.